src/cart/crt_internal.h

```diff
@@ -57,15 +57,16 @@
 	} while (0)
 
 /* Log an error with a RPC descriptor */
-#define RPC_ERROR(rpc, fmt, ...)					\
-	do {								\
-		D_TRACE_ERROR((rpc),					\
-			"[opc=%#x rpcid=%#lx rank:tag=%d:%d] " fmt,	\
-			(rpc)->crp_pub.cr_opc,				\
-			(rpc)->crp_req_hdr.cch_rpcid,			\
-			(rpc)->crp_pub.cr_ep.ep_rank,			\
-			(rpc)->crp_pub.cr_ep.ep_tag,			\
-			## __VA_ARGS__);				\
+#define RPC_ERROR(rpc, fmt, ...)						\
```

Comment:
(style)  line over 80 characters
(style)  Single statement macros should not use a do {} while (0) loop